### ISSUE TITLE
Explore how we can implement climate scenarios as a proto-DT extension

### ISSUE THREAD

**Issue by StefanVriend** [LTER-LIFE/VeluweProtoDT#42]
We will add a forecasting component to the proto-DT (see #41) to demonstrate the flexibility of adapting a workflow within the VRE. The first step is to explore how we can use the climate scenario work done by [Visser et al. 2021 (Proc R Soc B)](https://doi.org/10.1098/rspb.2021.1337) for our case.

**Comment by CherineJ** [comment]
The extension of the climate scenarios likely require two blocks:

-  Block 1: Validation of the scenarios: Before scenarios can be used, it should be tested whether they are actually sufficient to describe the measured biological data, for example by hindcasting and comparing predicted bud burst dates based on measured temperatures with the bud burst dates predicted by the scenario temperatures in the same time period.
-  Block 2: Use of the scenarios: Only if the validation shows that the scenario data is able to predict the biological data well, it can be used to forecast.

**Comment by CherineJ** [comment]
In the case of bud burst and the RCP 8.5 scenario, the validation of the scenario data shows that it deviates rather strongly from the measured KNMI temperatures and can therefore only poorly predict bud burst dates of the past. 
Marcel wants to ask Stephanie Jenouvrier for a meeting on this to discuss why there is such deviation in the temperature data and how to deal with this.

**Comment by CherineJ** [comment]
After deep diving into several scripts of the [initial work done on the scenarios studies and laying dates of birds ](https://royalsocietypublishing.org/doi/full/10.1098/rspb.2021.1337), we could construct a validation and a forecasting block for 5 different climate scenarios and the bud burst data. 

However, we encountered some hurdles along the way:

- Understand the scenario data: Each scenario contains several ensemble members/runs, each saved in a separate CSV file. Some of the file numbers are extended by the letter "c" and some runs do not contain missing years. As there is no extensive description for the files, it does not become clear where these differences come from. 

- z-Score transformation: The models can either be run on the actual temperatures or the zScores of temperatures.  It was not clear whether zScores of the historic period (1988 - 2023) are calculated the same way as for the future period (2024 - 2100) or if the later should be dependent on the mean of the historic period. We chose the later.
